src/util/data.js

```diff
@@ -333,3 +333,21 @@ export const overrideArrays = (objValue, srcValue, key, object, source, stack) =
     return srcValue;
   }
 };
+
+/**
+ * Humanizes a line item code. Strips the "line-item/" namespace
+ * definition from the beginnign, replaces dashes with spaces and
+ * capitalizes the first character.
+ *
+ * @param {string} code a line item code
+ *
+ * @return {string} returns the line item code humanized
+ */
+export const humanizeLineItemCode = code => {
+  if (!/^line-item\/.+/.test(code)) {
+    throw new Error(`Invalid line item code: ${code}`);
+  }
+  const lowercase = code.replace(/^line-item\//, '').replace(/-/g, ' ');
+
+  return lowercase.charAt(0).toUpperCase() + lowercase.slice(1);
```

Comment:
Btw. if clients can create line-items, can they create codes with format like `line-item/my-Banana-Inc-fee` and `line-item/myBananaIncFee`.

There's probably no need at this point address everything - I'm just not sure what is accepted as custom line-item by API?

Reply:
@Gnito the API takes anything up to 64 chars that starts with _line-item/_, so it can be myBananaIncFee too. Good remark, I could add a line item code format suggestion to the how-to guide.